src/reporter/tests/test_timex.py

```diff
@@ -64,13 +73,31 @@ def test_custom_index_takes_priority():
         select_time_index_value(headers, notification)
 
 
+def test_use_latest_meta_custom_index():
+    headers = {
+        TIME_INDEX_HEADER_NAME: 'customTimeIndex'
+    }
+    base_point = datetime(2019, 1, 1)
+
+    ts = build_notification_timepoints(base_point)
+    ts[0] = None  # custom index attribute slot
+
+    notification = build_notification(*ts)
+
+    # latest meta custom index slot = 8
+    assert ts[8] == \
+        select_time_index_value(headers, notification).isoformat()
+
+
 def test_skip_custom_index_if_it_has_no_value():
     headers = {
         TIME_INDEX_HEADER_NAME: 'customTimeIndex'
     }
     base_point = datetime(2019, 1, 1)
     ts = build_notification_timepoints(base_point)
     ts[0] = None  # custom index slot
+    ts[4] = None  # custom index metadata for a1
```

Comment:
How about defining have a bunch of module-level constants to represent these options and maybe a list containing all of them (or not) so we prevent confusions of indexing and duplicated comments? Something like what we see in `test_use_date_modified` but more visible.

Reply:
yah, them tests are a disaster area, sorry I whipped them together way too quickly...they should be rewritten properly...

Reply:
no worries! I just commented to check if @heikkilv , while at it, wouldn't mind doing this mini-change (only in this module at least) If not, we can do it later.

Reply:
I too think that the tests need a bigger rewrite. I don't think it is useful to modify them further with this pull request.